snap/snapcraft.yaml

```diff
@@ -54,6 +53,12 @@ slots:
     source:
       read:
         - $SNAP_COMMON/var/log/pgbouncer
+  sockets:
+    interface: content
+    content: socket-directory
+    source:
+      write:
+        - $SNAP_CURRENT/run
```

Comment:
There will be a directory structure underneath, we may tweak the path for ease of use.